src/test/java/com/redhat/cloud/policies/app/PolicyCrudServiceTest.java

```diff
@@ -65,44 +60,16 @@ void test() {
         assertEquals(5, history.getJsonObject("meta").getInteger("count"));
     }
 
-    private UUID createPolicy() {
+    @Transactional
```

Comment:
why is this needed now?

Reply:
Because of the SQL query that is executed in the new method body. The old version didn't execute a SQL query.